Kernel/CMakeLists.txt

```diff
@@ -687,9 +687,9 @@ add_compile_options(-fno-rtti -ffreestanding -fbuiltin)
 add_compile_options(-Wno-invalid-offsetof)
 
 if ("${SERENITY_ARCH}" STREQUAL "x86_64")
-    add_compile_options(-mno-80387 -mno-mmx -mno-sse -mno-sse2)
+add_compile_options(-mno-80387 -mno-mmx -mno-sse -mno-sse2)
 elseif("${SERENITY_ARCH}" STREQUAL "aarch64")
-    add_compile_options(-mgeneral-regs-only)
+add_compile_options(-mgeneral-regs-only)
 endif()
```

Comment:
Why did you dedent these lines? That doesn't seem related.
Same for the EFIPrekernel CMakeLists.txt.

Reply:
Not sure how that happened, and I had whitespace changes hidden on my end...